aten/src/ATen/Config.h.in

```diff
@@ -1,3 +1,5 @@
+// Copyright 2024 Arm Ltd. and/or its affiliates
+
```

Comment:
We usually don't have copyright headers in the individual files, especially in something autogenerated like  `.h.in` ones. I think https://github.com/pytorch/pytorch/pull/126687#issuecomment-2263607610 proposes an idea to add generic copyright header which will avoid need for individual changes like that

Reply:
ACK and Fixed